### ISSUE TITLE
[Keyboard Navigation - - Azure Databases Extension - Results tab]: Table cells present in the "Results" tab are not accessible with the arrow keys.

### ISSUE THREAD

**Issue by ChandiniSmiley** [microsoft/vscode-cosmosdb#2671]
### **User Experience:** 

Users who rely on keyboard for navigation will face difficulty in navigation as the user needs to navigate all the table controls to move to the next interactive control if table rows are not accessible with the arrow keys. 

**Note:** Login Credentials must not be provided within the bug. 

### **Pre-requisite:** 

Azure database should be installed from extensions in VS code. 

### **Repro Steps** 

1. Open VS Code and navigate to azure databases button present on right pane. 
2. Navigate to anyone of the resource and expand it. Expand a database. Right click on any collection and select "Open Query 
    editor(Preview)". 
3. Query editor screen appears. Run any sample query. Data in table appears under results tab. 
4. Navigate through all controls of the results tab including running the query. 
5. Verify if Table cells present in the "Results" tab are not accessible with the arrow keys. 

### **Actual Result:** 

Issue: Table cells present under the "Results" dialog are not accessible with the arrow keys. 

Observation: Complete table rows and each cells are only accessible with tab key due to which user needs to navigate all table rows to move to next controls.

### **Expected Result:** 

Table cells present under the "Results” tab should be accessible with the left/right arrow keys and on selecting tab key from entire row focus should move to next interactive control outside of the table.

### **MAS Reference:** 

[MAS 2.1.1 – Keyboard](https://aka.ms/MAS2.1.1)

### **Reference links:** 
 
- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries. 

### **Test Environment:**   

- OS Version: 24H2 (OS Build 26100.3476) 
- Visual Studio Code Version: 1.99.0 (user setup) 
- Tool: Accessibility Insights for Win32 
- Azure Databases for VS Code (Preview) Version : 0.25.2 

https://github.com/user-attachments/assets/33466658-5609-4494-afaa-8c7719ec9e1f

**Comment by ChandiniSmiley** [comment]
#A11yMAS;#A11ySev3;#Accessibility;#AILimited;#Benchmark;# GH-HCL-AzureDatabasesExtension-Apr2025;#HCL;#MAS2.1.1;#Win32;

**Comment by ChandiniSmiley** [comment]
@sevoku, can you please share the external team details which owns this control like ADO/ Service Tree ID so that we can route the bug to the correct team.
